### ISSUE TITLE
Code Lab Feedback

### ISSUE THREAD

**Issue by ackleyd1** [ackleyd1/codelab-twitter-bootstrap#1]
Hey @chrisvfritz, can you take a look at this? It's [hosted here](http://ravedave.surge.sh/) and meets the following [project](http://lansingcodelab.com/lessons/twitter-bootstrap/1) criteria:
- [x] include Bootstrap CSS and JS in an HTML document
- [x] use the `container` class appropriately to center your page content
- [x] split your content into 2 columns on large (lg) and medium (md) sized screens, but collapse them on top of each other on small (sm) and extra small (xs) screens
- [x] use a `thumbnail` class on an at least one image in your page
- [x] use a `navbar` class to display the name of your site and a list of at least 3 links at the top of your page

**Comment by chrisvfritz** [comment]
The way the Bootstrap's grid system works, you can actually shorten the classes on your columns from:

```
center col-xs-12 col-sm-16 col-md-6 col-lg-6
```

to:

```
center col-md-6
```

This is because the default for any size will be `col-*-12` (meaning they take up the entire grid width). So for `xs` and `sm`, these are already set up for you. When you set `col-md-6`, then it _also_ implies the same width for anything larger, just as if you had specified `col-lg-6`.

Does that make sense?

**Comment by ackleyd1** [comment]
Yes I think, so if i declare col-sm-4 for 3 columns of size 4 it will stay that way for sm, m, and lg screens but on xs they will be stacked on top of each other as size 12

**Comment by chrisvfritz** [comment]
Yes!

**Comment by chrisvfritz** [comment]
:shipit:
